codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/RuntimeType.kt

```diff
@@ -239,6 +239,10 @@ data class RuntimeType(val path: String, val dependency: RustDependency? = null)
         val ConstrainedTrait = RuntimeType("crate::constrained::Constrained", InlineDependency.constrained())
         val MaybeConstrained = RuntimeType("crate::constrained::MaybeConstrained", InlineDependency.constrained())
 
+        // serde types. They will be behind feature gates
```

Comment:
Avoid making comments about the future. All we can know is the present.

```suggestion
        // serde types. Gated behind `CfgUnstable`.
```

Reply:
OK.